compiler/daml-lf-ast/src/DA/Daml/LF/Ast/Pretty.hs

```diff
@@ -448,10 +467,14 @@ instance Pretty Expr where
       let (bs, e1) = view (rightSpine (unlocate _ETmLam)) e0
       in  hang (docTmLambda <> hsep (map (pPrintAndType lvl precParam) bs) <> docTmLambdaDot)
             2 (pPrintPrec lvl precELam e1)
-    e0@ETyLam{} -> maybeParens (prec > precELam) $
-      let (ts, e1) = view (rightSpine (unlocate _ETyLam)) e0
-      in  hang (docTyLambda <> hsep (map (pPrintAndKind lvl precParam) ts) <> docTyLambdaDot)
+    e0@ETyLam{}
```

Comment:
The description above mentions that you omit type annotations. Emitting type applications is another step imho. Not opposed to it but maybe update the comment.

Reply:
Good point. I should say we omit type information in level -2. It's not only type annotations but also type applications and type abstraction.